server/database-couch/helpers.js

```diff
@@ -1,15 +1,26 @@
 const axios = require('axios');
-const db = require('./schema.js');
+const nano = require('nano')('http://admin:123456@localhost:5984');
+const users3 = nano.use('users3');
 
-// const getUserById = async (userId) => {
-//   try {
-//     const user = await User.findOne({ userId });
-//     return user;
-//   } catch (err) {
-//     console.error(err);
-//     return null;
-//   }
-// };
+const dbname = 'users3';
```

Comment:
Maybe just name your database users instead of having a number at the end.

Reply:
I was playing with 3 different databases with different data in it. so ended up with 3 of them lol.